CodeChallenge/Challenges/Clock/Entries/BrandonShegaClockEntry.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+let brandonShegaClockEntry = CodeChallengeEntry<ClockChallenge>(name: "brandonshega") { input in
+  let df = DateFormatter()
+  df.dateFormat = "HH:MM:SS"
+  guard let date = df.date(from: input) else { return (0, 0, 0) }
+  let dateComps = Calendar.current.dateComponents([.hour, .minute, .second], from: date)
+  return dateComps.handsInAngles
+}
+
+extension DateComponents {
+  var hoursToAngle: Int {
+    guard let hour = hour else { return 0 }
+    return hour * 30
+  }
+
+  var minutesOrSecondsToAngle: Int {
```

Comment:
This name will pollute the global name space. Everything that isn't your actual entry should be marked `filePrivate`. Please update accordingly.